experimental/packages/opentelemetry-configuration/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@opentelemetry/configuration",
+  "version": "0.1",
+  "description": "OpenTelemetry Configuration",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
```

Comment:
I would like to have this package private until we actually use it somewhere. As soon as a package is available on npm, we get a lot of people wanting to use it. 😅